docs/API.md

```diff
@@ -44,11 +44,11 @@ s3Client = Minio('s3.amazonaws.com',
 ## 1. Constructor
 
 <a name="Minio"></a>
-### Minio(endpoint, access_key=None, secret_key=None, secure=True)
+### Minio(endpoint, access_key=None, secret_key=None, secure=True, region=None, http_client=None)
 
 |   |
 |---|
-| `Minio(endpoint, access_key=None, secret_key=None, secure=True, region=None, timeout=None)`  |
+| `Minio(endpoint, access_key=None, secret_key=None, secure=True, region=None, http_client=None)`  |
```

Comment:
We are breaking backward compatibility.  Is it expected?

Reply:
Yes @balamurugana - this is get python in line with minio-java

Reply:
then the next version needs to be updated in this PR